src/app/use-root-startup.ts

```
@@ -94,15 +96,14 @@ function startRootInitialization(
       refreshIsRTL = i18nModule.refreshIsRTL;
     } catch {
       // non-fatal
+    } finally {
+      if (!isMounted) return;
```

Biome (2.1.2)

[error] Unsafe usage of 'return'.

'return' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)